.envrc

```
@@ -2,7 +2,7 @@
 if [ ! -r .nix-disable  ] && has nix-env; then
   # set NIX_PROFILE so nix-env operations don't need to manually
   # specify the profile path
-  export NIX_PROFILE=/nix/var/nix/profiles/mymove-docs
+  export NIX_PROFILE="/nix/var/nix/profiles/per-user/${LOGNAME}/mymove-docs"
```

This works for me, but I wonder if we should have a check for whether or not the person has write permissions to the profile like we do in the mymove repo: 

https://github.com/transcom/mymove/blob/3661c4f0d2785d06307815374e020e109364f3df/.envrc#L302-L306

---

I'm with @felipe-lee on this. Let's standardize across the board. I never considered this project o need something like Nix, though I'm okay with it being included. But if we're gonna Nix let's Nix the way we do across the board. I'd strongly recommend we modify this to check and have error handling. 

```suggestion
  if [ -w "/nix/var/nix/profiles/per-user/${LOGNAME}" ]; then
    export NIX_PROFILE="/nix/var/nix/profiles/per-user/${LOGNAME}/mymove-docs"
  else
    log_error "Cannot determine writable location for your NIX_PROFILE"
  fi

```

---

I would be tempted to add it in but for the thread in the EngPlaybook where Drew was thinking it was clearer to allow the system to throw the error https://github.com/trussworks/Engineering-Playbook/pull/210#discussion_r767888609

---

does the system through the error properly? do we know what the error is? because this is a pre-message before Nix even runs versus what we're saying about having Nix throwing errors are different topics. But I'm not going to prevent this from making it into the repo. 👍🏽

---

Ah ok, sounds good